mysql-test/main/variables.result

```diff
@@ -971,6 +972,7 @@ SSL_CIPHER	#
 SSL_CRL	#
 SSL_CRLPATH	#
 SSL_KEY	#
+SSL_PASSPHRASE	#
```

Comment:
is it a good idea? I can imagine users objecting to having `SSL_PASSPHRASE` in `SHOW VARIABLES`

shall server remove the passphrase from argv after parsing?

when the server reads the passfile, before or after dropping the privileges?

Reply:
I remove the value in the latest patch, leaving variable name visible, but it will show value NULL, even if it was set in config file or command line.
I'm not sure we should remove the variable itself, it feels even more weird. if it is in the config file, but not mentioned in "show variables". People could at least get an idea of how variable is correctly spelled, and config file validators, if there are any outside, would continue to work.

Reply:
so it's always NULL whether it was set or not? May be set it to "file:xxx", "pass:xxx", "env:xxx" ? these could even be static hard-coded values, no need to generate them at run time. (or "file:" or "file:***" whatever)

Reply:
Fixed according to this proposal, i.e it would be "file:", "env:", or "pass:" if it was set.